services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -77,7 +108,11 @@ async def task_create_service_containers(
         await docker_compose_create(shared_store.compose_spec, settings)
 
         progress.publish(message="ensure containers are started", percent=0.95)
-        r = await docker_compose_start(shared_store.compose_spec, settings)
+
+        try:
+            r = await _retry_docker_compose_start(shared_store.compose_spec, settings)
```

Comment:
ok several questions here: 
1. Why are we adding this at all before we tested what really happens in the deployment?
2. if the creation fail you do not retry, why? I guess this could also happen
3. Why do we need to have the _RetryAgain exception here? isn't docker_compose_start already raising a specific exception when the command fails?
4. what is the rationale behind the 5 minutes? that is pretty long
5. do we need to care about cleaning out the containers? I guess this is taken care of when putting down the containers?